calico/tests/common.py

```diff
@@ -0,0 +1,47 @@
+EXTRA_METRICS = [
+    "felix_active_local_endpoints",
```

Comment:
same for tests

Reply:
I applied your suggestions to my code ! 
I also renamed metrics in my dashboard/monitor json, and ran all the test on my computer.